week4/tests.py

```diff
@@ -47,11 +47,22 @@ def find_lasers(path):
         return False
 
 
-def treat(path):
+def treat():
     url = ("https://raw.githubusercontent.com/"
            "notionparallax/code1161base/master/faces/")
-    name = path.split("/")[-1]
-    print("treat:", requests.get(url + name).text)
+
+    with open('.git/config', 'r') as f:
+        for line in f:
+            if 'code1161base' in line and not 'notionparallax' in line:
```

Comment:
The problem with this is that not all the repos are called code1161base.
I like the theory though.